proto/tripcommon.proto

```diff
@@ -150,6 +150,7 @@ message TaggedName {
   enum Type {
     kTunnel = 1;
     kBridge = 2;
+    kPronunciation = 3;
```

Comment:
if this is for street name pronunciation - then should it be stored with the street name it is associated with?